File: SplitStrings.js
solution('abc')

function solution(str){
    let stringPar = []
    var contPar = 0
   var cont = 0
   var i = 0
   for(i = 0; i < str.length; i++){
    cont++
    if(cont === 2){
        stringPar[contPar] = str[i-1] + str[i]
        contPar++
      cont = 0;
    }

   }
   if(str.length % 2 !== 0){
       stringPar[contPar] = str[str.length -1] + '_'
   }
   console.log(stringPar)
}

File: SplitStrings.py
def solution(s):
    stringPar = []
    cont = 0
    contPar = 0
    i = 0
    length = len(s)
    
    for i in range(len(s)):
        cont += 1
    
        if cont == 2:
            stringPar.append(s[i-1] + s[i])
            contPar += 1
            cont = 0

    if len(s) % 2 != 0:
        stringPar.append(s[len(s)-1] + '_')
    
    return stringPar
pass